books/RayTracingTheRestOfYourLife.html

```diff
@@ -332,8 +332,29 @@
         }
         ...
       private:
+        int    image_height;    // Rendered image height
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+        int    sqrt_spp;        // Square root of number of samples per pixel
+        double recip_sqrt_spp;  // 1 / sqrt_spp
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+        point3 center;          // Camera center
         ...
+        void initialize() {
+            ...
+            auto viewport_width = viewport_height * (double(image_width)/image_height);
+
+
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+            sqrt_spp = int(sqrt(samples_per_pixel));
+            recip_sqrt_spp = 1.0 / sqrt_spp;
+
+
```

Comment:
We don't change-highlight leading or trailing blank lines, even if added/removed.

Reply:
OK, I wasn't sure about this one. I'll fix it in a follow-up.